pyqtgraph/graphicsItems/AxisItem.py

```diff
@@ -1070,7 +1068,7 @@ def generateDrawSpecs(self, p):
                 if s is None:
                     rects.append(None)
                 else:
-                    br = p.boundingRect(QtCore.QRectF(0, 0, 100, 100), QtCore.Qt.AlignmentFlag.AlignCenter, asUnicode(s))
+                    br = p.boundingRect(QtCore.QRectF(0, 0, 100, 100), QtCore.Qt.AlignmentFlag.AlignCenter, str(s))
```

Comment:
`strings` can either be `None` or of type `str`, since we already filter out the `None` we can ditch this casting too.